File: Selenium/Activity13.java
package Selenium;

import java.util.List;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity13 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		driver.manage().window().maximize();
		driver.get("https://training-support.net/webelements/tables");
		System.out.println("Page Title is : " + driver.getTitle());
		List <WebElement> row = driver.findElements(By.xpath("//table[contains(@class, 'able-auto')]/tbody/tr"));
		List <WebElement> col = driver.findElements(By.xpath("//table[contains(@class, 'able-auto')]/tbody/tr/td"));
		System.out.println("Number of rows are : "+ row.size());
		System.out.println("Number of cols are : "+ col.size());
		List <WebElement> tRow = driver.findElements(By.xpath("//table[contains(@class, 'able-auto')]/tbody/tr[3]"));
		for(WebElement thirdRow : tRow) {
			System.out.println("3rd Row Data : " + thirdRow.getText());
		}
		String cellVal = driver.findElement(By.xpath("//table[contains(@class, 'able-auto')]/tbody/tr[2]/td[2]")).getText();
		System.out.println("2nd cell for 2nd row val is : " + cellVal);
		driver.quit();
	}

}

File: Selenium/Activity13.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Chrome() as driver:
    wait = WebDriverWait(driver, timeout=10)
    driver.get("https://training-support.net/webelements/tables")
    print(driver.title)
    rows = driver.find_elements(By.XPATH, "//table[contains(@class, 'able-auto')]/tbody/tr")
    cols = driver.find_elements(By.XPATH, "//table[contains(@class, 'able-auto')]/tbody/tr/td")
    print("Number of rows in table : ", len(rows))
    print("Number of rows in table : ", len(cols))
    thirdRow = driver.find_elements(By.XPATH, "//table[contains(@class, 'able-auto')]/tbody/tr[3]")
    for row in thirdRow:
        print(row.text)
    cells = driver.find_element(By.XPATH, "//table[contains(@class, 'able-auto')]/tbody/tr[2]/td[2]")
    print(cells.text)
    driver.quit()